### ISSUE TITLE
Grouping for fields in admin interface?

### ISSUE THREAD

**Issue by qrilka** [jaredly/django-appsettings#3]
Any plans for creating grouping in setings admin?
Separate forms for separate registered settings classes?
Seems to be quite easy (I'm a newbie on github yet so need to learn it to make my own forks to make contribution easier)

**Comment by jaredly** [comment]
You're right, that would be quite easy to implement. I'm in the middle of writing a paper at the moment, but I'd be happy to accept a patch (or pull request). Otherwise, I'll probably get to it myself some time next week.

here's a great intro to forking: http://help.github.com/forking/

**Comment by stormlifter** [comment]
How'd that paper go? You still alive? I really like how you wrote this app. I think I may fork it when I get some time.
